src/pybind/mgr/dashboard_v2/frontend/src/app/ceph/rgw/rgw-daemon-details/rgw-daemon-details.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { RgwDaemonService } from '../services/rgw-daemon.service';
```

Comment:
please add a empty line between this 3 lines and reorder them as @angular, lodash and then service.
Maybe you can remove the tslint disable after.